src/components/map.jsx

```diff
@@ -41,12 +41,12 @@ const AsyncMap = compose(
 		onClick={props.setMarker}
 		options={{ mapTypeControl: false, streetViewControl: false }}
 	>
-		<Marker
+		{(props.displayMarker !== '') ? <Marker
```

Comment:
my guess but i don't know for sure is that this should check

```js
{(props.displayMarker ? ... : null)}
```

this will also make sure displayMarker is not null or undefined

Reply:
updated